### PR TITLE
Add PREFIX_VOLUME and change README

### LINKED ISSUE #8: Add the possibility to change the location of volumes on host
By default, the location of volumes on host is `/srv`, but in some cases Docker may complain
```
docker: Error response from daemon: error while creating mount source path '/srv/mongodb': mkdir /srv/mongodb: read-only file system.
```
It's necessary to add a new parameter to cusdeb-single-node which will allow to change the location.

### PR DESCRIPTION
**Pull request by mdanil007**
close #8

### CODE REVIEW

File: functions.sh [hunk 1]
```diff
@@ -181,14 +181,14 @@ run_containers() {
     # MongoDB
     #
     docker pull mongo:"${MONGO_TAG}"
-    docker run --name cusdeb-mongo -v /srv/mongodb:/data/db -p "${MONGO_PORT}":27017 -d mongo:"${MONGO_TAG}"
+    docker run --name cusdeb-mongo -v "${VOLUME_PREFIX}"/mongodb:/data/db -p "${MONGO_PORT}":27017 -d mongo:"${MONGO_TAG}"
     wait_for "${MONGO_PORT}"
-
+
```
Comment (eugulixes) [inline functions.sh]: Redundant spaces.

File: single-node.sh [hunk 2]
```diff
@@ -69,6 +69,8 @@ export PG_TAG=${PG_TAG:=9.4}
 
 export PG_USER=${PG_USER:=postgres}
 
+export VOLUME_PREFIX=${VOLUME_PREFIX:=/srv}
```
Comment (eugulixes) [inline single-node.sh]: Do not break the alphabetical order of the variables.

File: README.md [hunk 3]
```diff
@@ -16,3 +16,15 @@ Build project in directory ../cusdeb.
 sudo ./single-node.sh build ../cusdeb
 ```
 Project depends on some code which is not yet published in GitHub
+
```
Comment (eugulixes) [inline README.md:19]: There must be a section called, for example, `Troubleshooting`.
```
## Troubleshooting
```

File: README.md [hunk 4]
```diff
@@ -16,3 +16,15 @@ Build project in directory ../cusdeb.
...
+If you encounter the error:
```
Comment (eugulixes) [inline README.md]:
```
If you executed the `start` command and encountered the following error
```

File: README.md [hunk 5]
```diff
@@ -16,3 +16,15 @@ Build project in directory ../cusdeb.
...
+
+```
+docker: Error response from daemon: error while creating mount source path '/srv/mongodb': mkdir /srv/mongodb: read-only file system.
+```
+
+Then you need to change the variable:
```
Comment (eugulixes) [inline README.md]:
```
try to change the default value of the `VOLUME_PREFIX` environment variable.
```

**Review by eugulixes — CHANGES_REQUESTED** [review]

**Review by eugulixes — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -16,3 +16,13 @@ Build project in directory ../cusdeb.
...
+
+## Troubleshooting
+
+If you executed the `start` command and encountered the following error
+
+```
+docker: Error response from daemon: error while creating mount source path '/srv/mongodb': mkdir /srv/mongodb: read-only file system.
+```
+
+try to change the default value of the `VOLUME_PREFIX` environment variable.
```
Comment (denismosolov) [inline README.md:28]: How about 'try to change the value of the `VOLUME_PREFIX` to something inside your user's home directory'?

Comment (eugulixes) [reply README.md:28]: It seems to me like overcomplicating.

**Pull request merged**